src/pages/PrefectureTable/index.js

```diff
@@ -0,0 +1,198 @@
+import * as c3 from "c3";
+import _ from "lodash";
```

Comment:
Instead of importing all `lodash` methods it'll be better to import only what is actually needed